apps/swirl-docs/src/components/Layout/CategoryNav.tsx

```diff
@@ -51,7 +51,7 @@ export const CategoryNav: FunctionComponent<CategoryNavProps> = ({
   return (
     <nav
       aria-label="category"
```

Comment:
**issue:** I think this needs to be adjustable. For the components overview "category" is not the right label.

Reply:
yeah, good point.